src/yadmark/benchmark/external/apfelpy_utils.py

```diff
@@ -226,55 +287,42 @@ def compute_apfelpy_data(theory, observables, pdf):
         theory["mb"] * theory["kbThr"],
         theory["mt"] * theory["ktThr"],
     ]
+    masses = [
+        0,
+        0,
+        0,
+        theory["mc"],
+        theory["mb"],
+        theory["mt"],
+    ]
 
     # Setting the theory
     fns = theory["FNS"]
-    if fns != "ZM-VFNS":
-        raise ValueError("APFEL++ benchmark contains only ZM-VFNS.")
+    if fns not in ["ZM-VFNS", "FFNS", "FFNS0"]:
+        raise ValueError(f"APFEL++ does not contain {fns}.")
 
     # Perturbative Order
     pto = theory["PTO"]
 
     # Couplings
-    alphas = ap.AlphaQCD(theory["alphas"], theory["Qref"], thrs, pto)
+    alphas = ap.AlphaQCD(theory["alphas"], theory["Qref"], masses, thrs, pto)
     alphas = ap.TabulateObject(alphas, 2 * NQ, QMin, QMax, 3)
 
     # Map Yadism observables to Apfel++ Objects
-    init = ap.initializers
-    if observables["prDIS"] == "CC":
-        projectile_pids = {
-            "electron": 11,
-            "positron": -11,
-            "neutrino": 12,
-            "antineutrino": -12,
-        }
-        if projectile_pids[observables["ProjectileDIS"]] > 0:
```

Comment:
@Radonirinaunimi if `projectile_pids` do we crash here right ?

Reply:
Sorry for the late reply! It doesn't come to me right away why exactly this was just passed previously (as opposed to not forced to raise an exception), to be fully certain I will test this tomorrow and check (I will also review this PR alongside).

Reply:
I have looked into this and now everything comes back, to be more exact in the error raised I'd print out the following message instead as the current one implies that it is not possible to get it from `APFEL++`:
```python
raise ValueError(f"Benchmarks not implemented for CC {observables['ProjectileDIS']} ")
```